### PR TITLE
[13.0][MIG] hr_employee_phone_extension

### PR DESCRIPTION
**Pull request by Saran440**
Change fields position to tab `Work Information`
I think it look good.
Selection_003

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: new position is ok to me.

**Review by lfreeke — APPROVED** [review]
Functional test :+1:

**Review by etobella — APPROVED** [review]

**Comment by newtratip** [comment]
@Saran440 Conflict file .isort.cfg

File: .isort.cfg [hunk 1]
```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=
+known_third_party=setuptools
```
Comment (newtratip) [inline .isort.cfg]:
```suggestion
known_third_party=dateutil,setuptools
```

Comment (Saran440) [reply .isort.cfg]: Fixed. Could you check again? Thank you :)

**Review by newtratip — APPROVED** [review]
Functional test :+1:

**Pull request merged**